flake.nix

```diff
@@ -0,0 +1,42 @@
+{
+  description = "virtual environments";
+
+  inputs = {
+    devshell.url = "github:numtide/devshell";
+    flake-utils.url = "github:numtide/flake-utils";
+    unstableNixPkgs.url = "nixpkgs/nixos-unstable";
```

Comment:
What's this? Why is it unstable?

Reply:
I was using yarn from the unstable nix channel as the latest version didn't have yarn v2 available

It's now available, so we don't need to fetch it from the unstable channel

updated in f5fa3d18ad6aa71a3236dee14ac6aeb8a4a2f6fd